manifests/backend.pp

```diff
@@ -136,6 +138,7 @@
   Hash $roles = {},
   Hash $silencings = {},
   Hash $users = {},
+  Optional[Integer] $sensuctl_chunk_size = undef,
```

Comment:
should we be more specific with regards to the integer? do the docs give a valid range?

Reply:
Nothing in docs about a range, I would assume it's 0 to some large number.

https://docs.sensu.io/sensu-go/5.8/sensuctl/reference/#handling-large-datasets

Reply:
I just tested in Vagrant, `--chunk-size` takes negative numbers and 0 as well as expected positive integers.  All returned all resources. I am inclined not to impose a range if sensuctl isn't imposing one.

Reply:
Looks like any integer is valid - https://github.com/sensu/sensu-go/search?q=ChunkSize&unscoped_q=ChunkSize